app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -115,15 +116,23 @@ def export
       def generate
         @composer = ReportComposer.from_api_params(params)
         if @composer.valid?
-          response = @composer.render(params: params)
-          send_data response, type: @composer.mime_type, filename: @composer.report_filename
+          response.headers['Content-Type'] = @composer.mime_type
```

Comment:
https://edgeapi.rubyonrails.org/classes/ActionController/Live.html suggests it should be `text/event-stream`